lib/services/northBound/restUtils.js

```diff
@@ -150,6 +207,12 @@ function isTimestamped(payload) {
     return false;
 }
 
+/**
+ * Checks if timestamp attributes are included in NGSIv1 entities.
+ *
+ * @param {Object} payload         NGSIv1 payload to be analyzed.
+ * @return {Boolean}               true if timestamp attributes are included. false if not.
+ */
 function isTimestampedNgsi2(payload) {
```

Comment:
JavaDoc says "Checks if timestamp attributes are included in NGSIv1 entities" but name of the function mentions NGSIv2...

Is this correct?